定数の初期化を修正する

# 目的 : Must

UnitTestを実行できるようにする
## 背景

``` shell
../hrp2/sdk/workspace/hackEV/sensors/SonarSensorController.h:36:57: warning: non-static data member initializers only available with -std=c++11 or -std=gnu++11 [enabled by default]
     const int16_t SAFETY_CATCHABLE_PRIZE_DISTANCE_MIN = 3;
                                                         ^
../hrp2/sdk/workspace/hackEV/sensors/SonarSensorController.h:38:57: warning: non-static data member initializers only available with -std=c++11 or -std=gnu++11 [enabled by default]
     const int16_t SAFETY_CATCHABLE_PRIZE_DISTANCE_MAX = 3;
```
# やる事 : Must
- [ ] 上述のエラーを修正する
# 考慮した事
## [リスク](https://kotobank.jp/word/%E3%83%AA%E3%82%B9%E3%82%AF-183705)

なし
## [懸念事項](https://kotobank.jp/word/%E6%87%B8%E5%BF%B5-490895)

なし

--- a/hrp2/sdk/workspace/hackEV/sensors/SonarSensorController.cpp
+++ b/hrp2/sdk/workspace/hackEV/sensors/SonarSensorController.cpp
@@ -10,6 +10,8 @@ SonarSensorController::SonarSensorController(sensor_port_t _port)
     : port(_port)
     , enabled(false)
     , initialized(false)
+    , SAFETY_CATCHABLE_PRIZE_DISTANCE_MIN(3)
+    , SAFETY_CATCHABLE_PRIZE_DISTANCE_MAX(3)
 {
 }
 /**

--- a/hrp2/sdk/workspace/hackEV/sensors/SonarSensorController.h
+++ b/hrp2/sdk/workspace/hackEV/sensors/SonarSensorController.h
@@ -33,7 +33,8 @@ private:
     bool    initialized;
 
     //! 検証を安全につかめる最小距離
-    const int16_t SAFETY_CATCHABLE_PRIZE_DISTANCE_MIN = 3;
+    const int16_t SAFETY_CATCHABLE_PRIZE_DISTANCE_MIN;
+
     //! 検証を安全につかめる最大距離
-    const int16_t SAFETY_CATCHABLE_PRIZE_DISTANCE_MAX = 3;
+    const int16_t SAFETY_CATCHABLE_PRIZE_DISTANCE_MAX;
 };

